pandas/tests/frame/indexing/test_insert.py

```diff
@@ -0,0 +1,68 @@
+"""
```

Comment:
shouldn't also move all column setitem tests here?

Reply:
current plan is for test_setitem file(s)